src/sql/base/browser/ui/infoButton/infoButton.css

```diff
@@ -37,37 +37,9 @@
 }
 
 .info-button {
-	background-color: #fff;
 	border-radius: 2px;
-	box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.13);
```

Comment:
All these box shadow values you removed are actually part of the design per the Figma comps. Please make sure they remain.